### ISSUE TITLE
Incorrect accessibility computation of `dialog`

### ISSUE THREAD

**Issue by kagankan** [markuplint/markuplint#796]
When I hover `<dialog>` element,

```html
<dialog></dialog>
```

popup shows below:

```
<dialog> Computed Accessibility Properties:

No exposed to accessibility tree (hidden element)
```

The `dialog` element without `open` attribute is indeed not displayed on most browsers, but that behavior is due to CSS.

# Expected behavior

The popup shows accesibility properties about the `dialog` element.

**Comment by YusukeHirao** [comment]
Resolved in v4.
